### PR TITLE
KiCAD 7 CLI Update

### PR DESCRIPTION
**Pull request by ayushchakra**

### CODE REVIEW

File: test
Comment (ayushchakra) [inline test:1]: what is this

Comment (tlee10333) [reply test:1]: Accidental file that got caught up in the commit. I deleted it though

Comment (tlee10333) [reply test:1]: It's a blank file @redhotchili0909 made. idk why it existed but Ig I deleted it?

Comment (ayushchakra) [reply test:1]: yeah delete it

**Review by amit-kumarh — APPROVED** [review]
LGTM. Thank you for your work on this!

**Pull request merged**